draft-ietf-sframe-enc.md

```diff
@@ -149,60 +147,72 @@ media in a broad range of scenarios, as outlined by the following goals:
    scenarios.
 
 
-#SFrame
-We propose a frame level encryption mechanism that provides effective end-to-end encryption, is simple to implement, has no dependencies on RTP, and minimizes encryption bandwidth overhead. Because SFrame encrypts the full frame, rather than individual packets, bandwidth overhead is reduced by having a single IV and authentication tag for each media frame.
+# SFrame
+
+This document defines an encryption mechanism that provides effective end-to-end encryption, is simple to implement, has no dependencies on RTP, and minimizes encryption bandwidth overhead. Because SFrame can encrypt a full frame, rather than individual packets, bandwidth overhead can reduced by adding encryption overhead only once per media frame, instead of once per packet.
```

Comment:
This is pre-existing, but we should probably wrap these lines by 80 characters like you are doing below.